applications/nrf5340_audio/Kconfig.sysbuild

```diff
@@ -4,7 +4,37 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
-
 config NRF_DEFAULT_IPC_RADIO
 	default y
+
+config AUDIO_DFU_EXTERNAL_FLASH
+	bool "Audio Application DFU using external flash"
+	default n
+	help
+	  Use external flash to store the images downloaded during DFU. This requires the external
+	  flash shield with the MX25R6435F on the SPI bus.
+
+config AUDIO_DFU_INTERNAL_FLASH
+	bool "Audio Application DFU using internal flash"
+	default n
```

Comment:
Should there be some `depends on` here? Both of these cannot be set to "y" at the same time.